docs/README.md

```diff
@@ -0,0 +1,30 @@
+# Optimizador de Restaurantes
+
+## Definición del Problema
+
+### Definición del Problema General
+
+Tiempo medio que se espera entre que se pide una comanda en un bar o restaurante y el tiempo en que se reciben las bebidas, platos o tapas y postres de dicha comanda.
```

Comment:
Os he repetido varias veces que los cambios que no sean evaluables no lo metáis en el PR. Hacedlo antes del PR. Esto sólo añade ruido y trabajo a quien revisa.

Reply:
Ahora mismo, estoy haciendo todos los cambios a nivel de PR.

Si veo en este caso que no tiene nada que ver con el objetivo, ¿puedo crear una PR y mergearla sin necesidad de ponerlo en los objetivos?
Dicho de otra manera, puedo hacer cambios en mis proyectos mientras que no tengan relación con los objetivos.

En este caso la reorganización del README.md no lo tiene como dices.

Reply:
Es lo que te estoy diciendo. Para evaluar un objetivo, lo mejor es que pongáis en el PR sólo lo necesario para evaluarlo.